hadoop-cloud/src/main/scala/org/apache/spark/internal/io/cloud/BindingParquetOutputCommitter.scala

```diff
@@ -104,8 +105,9 @@ class BindingParquetOutputCommitter(
       committer.abortJob(jobContext, state)
     } catch {
       case e: IOException =>
-        // swallow exception to avoid problems when called within exception
-        // handlers
+        /** 
+        * swallow exception to avoid problems when called within 
+        * exception handlers **/ 
```

Comment:
The format here is wrong, and I think we won't need the comment on the top - we wouldn't comment individual import statements.